change/@ni-nimble-components-ceebd404-0fb3-43e5-8f98-04f4eed64531.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
It looks like we are currently testing combinations that we don't care about -- particularly Focused + Disabled. I wouldn't expect our disabled controls to be focusable, so it seems like we'd be adding extra noise in our chromatic diffs if those states change. Can you think of a good way to skip a state like that?

Reply:
One option would be to split the interaction states between two stories, where one story would include disabled states with just hover interactions, and the other would include other interactions without disabling. But that creates more stories and doesn't scale well.

Instead, I added an optional `filter` parameter to `createMatrix` which can be used to exclude any unwanted combination of states from the rendered story. For the interaction stories, I can pass a simple filter function that excludes all disabled configs, except if the interaction state is hover (i.e. not active or focused).

There are possibly cases in other non-interaction visual tests that we would want to exclude, too. For example, I added a filter to the menu button matrix test to exclude open + disabled combinations, since that's not really a realistic state.